src/starkware/cairo/lang/compiler/ast/code_elements.py

```diff
@@ -586,6 +587,31 @@ def get_children(self) -> Sequence[Optional[AstNode]]:
         return [self.condition, self.main_code_block, self.else_code_block]
 
 
+@dataclasses.dataclass
+class CodeElementFor(CodeElement):
```

Comment:
I imagine this is not yet handled down the line (e.g. preprocessor), right? I want it to fail with a good error message. You should probably implement it there and raise PreprocessorError("For loops are not supported yet."), or something similar,